PyInstaller/hooks/hook-PyQt5.QtWebEngineWidgets.py

```diff
@@ -21,17 +21,30 @@
 if compat.is_darwin:
     # This is based on the layout of the Mac wheel from PyPi.
     data_path = pyqt5_library_info.location['DataPath']
+    web_engine_process = ['lib', 'QtWebEngineCore.framework', 'Helpers', 'QtWebEngineProcess.app']
     resources = ['lib', 'QtWebEngineCore.framework', 'Resources']
-    web_engine_process = ['lib', 'QtWebEngineCore.framework', 'Helpers']
+    # These resource files for QtWebEngineProcess.app must located in .app/Content/Resources
+    # see https://doc.qt.io/qt-5.10/qtwebengine-deploying.html
+    res_files = ['qtwebengine_resources.pak',  # contains the resources needed by Chromium.
+                 'qtwebengine_devtools_resources.pak',  # contains tools for remote debugging.
+                 'qtwebengine_resources_100p.pak',  # contains images suitable for low resolution displays.
+                 'qtwebengine_resources_200p.pak',  # contains images suitable for high DPI displays.
+                 'icudtl.dat',
+                 ]
     # When Python 3.4 goes EOL (see
     # `PEP 448 <https://www.python.org/dev/peps/pep-0448/>`_, this is
     # better written as ``os.path.join(*rel_data_path, *resources[:-1])``.
     datas += collect_system_data_files(
         os.path.join(data_path, *resources),
         os.path.join(*(rel_data_path + resources[:-1])), True)
+    for r in res_files:
+        source = os.path.join(data_path, *(resources + [r]))
+        dest = '.'
```

Comment:
Do we really need this variable? IMHO it does not add to the understanding of the code.